### ISSUE TITLE
Sudden increase of training loss after validation step

### ISSUE THREAD

**Issue by weirayao** [kuleshov-group/bd3lms#24]
As seen below, I was running the training script with and without loading the pretrained ckpt `kuleshov-group/bd3lm-owt-block_size1024-pretrain`. 
```bash
sbatch scripts/train/train_owt_bd3lm.sh
```
The red curve for training from scratch, the loss increased suddenly after validation step at 2.5k. Any idea why this happens? Thanks! 

<img width="552" alt="Image" src="https://github.com/user-attachments/assets/63fd5b61-3ec0-4d04-8f2f-92b8d92848f0" />

**Comment by weirayao** [comment]
I see. That explains everything. Thanks @mariannearr !
